ibc-core/ics02-client/types/src/events.rs

```diff
@@ -236,14 +229,18 @@ impl TryFrom<abci::EventAttribute> for ConsensusHeightsAttribute {
                 let consensus_heights: Vec<Height> = value
                     .split(',')
                     .map(|height_str| {
-                        Height::from_str(height_str)
-                            .map_err(|_| ClientError::InvalidAttributeValue(height_str.to_string()))
+                        Height::from_str(height_str).map_err(|e| {
```

Comment:
I think it makes sense for use `DecodingError` for `Height`'s `FromStr` and `TryFrom` conversions.